### PR TITLE
Ibm qradar plugin 1 0 0

### PR DESCRIPTION
**Pull request by hiren-sacumen**
### Description

  - Change the toll tip description in the input fields for all actions and trigger.

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [x] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

New Plugin Uploaded.

image

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [x] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [x] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [x] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [x] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [x] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [x] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [x] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [x] Work fully completed
- [x] Functional
  - [x] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [x] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)

    ```
      INFO[0000] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug test < tests/add_notes_to_offense.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: add_notes_to_offense
      {
        "connection": "successful"
      }
      
      INFO[0001] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug test < tests/get_ariel_search_by_id.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_ariel_search_by_id
      {
        "connection": "successful"
      }
      
      INFO[0002] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug test < tests/get_assets.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_assets
      {
        "connection": "successful"
      }
      
      INFO[0004] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug test < tests/get_offense_closing_reasons.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_offense_closing_reasons
      {
        "connection": "successful"
      }
      
      INFO[0005] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug test < tests/get_offense_note.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_offense_note
      {
        "connection": "successful"
      }
      
      INFO[0006] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug test < tests/get_offense_note_by_id.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_offense_note_by_id
      {
        "connection": "successful"
      }
      
      INFO[0008] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug test < tests/get_offenses.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_offenses
      {
        "connection": "successful"
      }
      
      INFO[0009] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug test < tests/start_ariel_search.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: start_ariel_search
      {
        "connection": "successful"
      }
      
      INFO[0010] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug test < tests/update_offense.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: update_offense
      {
        "connection": "successful"
      }

    ```

  - [x] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)
    ```
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: add_notes_to_offense
      Offense ID provided: 33
      Note Text provided: New Note text
      Final Url:
      {
        "data": {
          "create_time": 1641908199188,
          "id": 353,
          "note_text": "New Note text",
          "username": "API_user: admin"
        }
      }
      
      INFO[0002] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug run < tests/get_ariel_search_by_id.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_ariel_search_by_id
      Search ID provided: 30562a91-2b1f-4672-a7c1-d2e33bcabc1b
      Poll Interval Provided: 0
      {
        "data": {
          "completed": true,
          "compressed_data_file_count": 0,
          "compressed_data_total_size": 0,
          "cursor_id": "30562a91-2b1f-4672-a7c1-d2e33bcabc1b",
          "data_file_count": 6,
          "data_total_size": 1622519,
          "desired_retention_time_msec": 86400000,
          "index_file_count": 0,
          "index_total_size": 0,
          "processed_record_count": 150139,
          "progress": 100,
          "progress_details": [],
          "query_execution_time": 255,
          "query_string": "SELECT * FROM events LAST 10 MINUTES",
          "record_count": 150139,
          "save_results": false,
          "search_id": "30562a91-2b1f-4672-a7c1-d2e33bcabc1b",
          "size_on_disk": 48343406,
          "status": "COMPLETED",
          "subsearch_ids": []
        }
      }
      
      INFO[0005] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug run < tests/get_assets.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_assets
      {
        "data": [
          {
            "domain_id": 0,
            "hostnames": [],
            "id": 0,
            "interfaces": [],
            "products": [],
            "properties": [],
            "risk_score_sum": 0,
            "users": [],
            "vulnerability_count": 0
          },
          {
            "domain_id": 0,
            "hostnames": [
              {
                "created": 1640582726381,
                "id": 1001,
                "name": "example.com",
                "type": "DNS"
              }
            ],
            "id": 1001,
            "interfaces": [
              {
                "created": 1640264696400,
                "id": 1001,
                "ip_addresses": [
                  {
                    "created": 1640264696400,
                    "id": 1001,
                    "network_id": 2,
                    "type": "IPV4",
                    "value": "192.10.10.178"
                  }
                ]
              }
            ],
            "products": [
              {
                "id": 1000,
                "product_variant_id": 40335
              }
            ],
            "properties": [
              {
                "id": 1000,
                "last_reported": 1640264696400,
                "last_reported_by": "USER:admin",
                "name": "Given Name",
                "type_id": 1001,
                "value": "TEst"
              },
              {
                "id": 1001,
                "last_reported": 1640264698711,
                "last_reported_by": "USER:admin",
                "name": "Unified Name",
                "type_id": 1002,
                "value": "TEst"
              },
              {
                "id": 1002,
                "last_reported": 1640264698711,
                "last_reported_by": "USER:admin",
                "name": "Primary OS ID",
                "type_id": 1033,
                "value": "40335"
              }
            ],
            "risk_score_sum": 0,
            "users": [],
            "vulnerability_count": 0
          },
          {
            "domain_id": 0,
            "hostnames": [],
            "id": 1002,
            "interfaces": [
              {
                "created": 1640265476716,
                "id": 1002,
                "ip_addresses": [
                  {
                    "created": 1640265476716,
                    "id": 1002,
                    "network_id": 2,
                    "type": "IPV4",
                    "value": "192.10.10.179"
                  }
                ]
              }
            ],
            "products": [],
            "properties": [
              {
                "id": 1003,
                "last_reported": 1640265476727,
                "last_reported_by": "USER:admin",
                "name": "Unified Name",
                "type_id": 1002,
                "value": "192.10.10.179"
              }
            ],
            "risk_score_sum": 0,
            "users": [],
            "vulnerability_count": 0
          }
        ]
      }
      
      INFO[0007] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug run < tests/get_offense_closing_reasons.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_offense_closing_reasons
      Include_deleted provided: False
      Include_reserved provided: False
      {
        "data": [
          {
            "id": 1,
            "is_deleted": false,
            "is_reserved": false,
            "text": "Non-Issue"
          },
          {
            "id": 3,
            "is_deleted": false,
            "is_reserved": false,
            "text": "Policy Violation"
          }
        ]
      }
      
      INFO[0010] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug run < tests/get_offense_note.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_offense_note
      Offense Id provided: 33
      {
        "data": [
          {
            "create_time": 1641451278329,
            "id": 209,
            "note_text": "New Note text",
            "username": "API_user: admin"
          },
          {
            "create_time": 1641466912789,
            "id": 216,
            "note_text": "New Note text",
            "username": "API_user: admin"
          }
        ]
      }
      
      INFO[0012] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug run < tests/get_offense_note_by_id.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_offense_note_by_id
      Offense Id provided: 33
      Note Id provided: 33
      {
        "data": {
          "create_time": 1640936459993,
          "id": 51,
          "note_text": "Test ",
          "username": "API_user: admin"
        }
      }
      
      INFO[0014] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug run < tests/get_offenses.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: get_offenses
      {
        "data": [
          {
            "categories": [
              "Access Denied"
            ],
            "category_count": 1,
            "credibility": 3,
            "description": "Access Denied\n",
            "destination_networks": [
              "Net-10-172-192.Net_172_16_0_0",
              "other"
            ],
            "device_count": 1,
            "domain_id": 0,
            "event_count": 308,
            "first_persisted_time": 1641359712000,
            "flow_count": 0,
            "follow_up": false,
            "id": 39,
            "inactive": true,
            "last_persisted_time": 1641833013000,
            "last_updated_time": 1641381314500,
            "local_destination_address_ids": [
              14
            ],
            "local_destination_count": 1,
            "log_sources": [
              {
                "id": 63,
                "name": "Custom Rule Engine-8 :: ip-172-31-34-93",
                "type_id": 18,
                "type_name": "EventCRE"
              }
            ],
            "magnitude": 2,
            "offense_source": "169.254.3.3",
            "offense_type": 0,
            "policy_category_count": 0,
            "protected": false,
            "relevance": 0,
            "remote_destination_count": 1,
            "rules": [
              {
                "id": 100425,
                "type": "CRE_RULE"
              }
            ],
            "security_category_count": 1,
            "severity": 6,
            "source_address_ids": [
              20
            ],
            "source_count": 1,
            "source_network": "other",
            "start_time": 1641359711660,
            "status": "OPEN",
            "username_count": 2
          },
          {
            "categories": [
              "Access Denied",
              "ACL Deny"
            ],
            "category_count": 2,
            "credibility": 2,
            "description": "Excessive Firewall Denies Between Hosts\n",
            "destination_networks": [
              "Net-10-172-192.Net_172_16_0_0",
              "other"
            ],
            "device_count": 1,
            "domain_id": 0,
            "event_count": 119804248,
            "first_persisted_time": 1640866551000,
            "flow_count": 0,
            "follow_up": true,
            "id": 38,
            "inactive": false,
            "last_persisted_time": 1641908186000,
            "last_updated_time": 1641908182050,
            "local_destination_address_ids": [
              14
            ],
            "local_destination_count": 1,
            "log_sources": [
              {
                "id": 63,
                "name": "Custom Rule Engine-8 :: ip-172-31-34-93",
                "type_id": 18,
                "type_name": "EventCRE"
              }
            ],
            "magnitude": 4,
            "offense_source": "172.31.34.93",
            "offense_type": 0,
            "policy_category_count": 0,
            "protected": true,
            "relevance": 5,
            "remote_destination_count": 1,
            "rules": [
              {
                "id": 100575,
                "type": "CRE_RULE"
              },
              {
                "id": 100033,
                "type": "CRE_RULE"
              },
              {
                "id": 100376,
                "type": "CRE_RULE"
              },
              {
                "id": 100426,
                "type": "CRE_RULE"
              },
              {
                "id": 100475,
                "type": "CRE_RULE"
              }
            ],
            "security_category_count": 2,
            "severity": 5,
            "source_address_ids": [
              19
            ],
            "source_count": 1,
            "source_network": "Net-10-172-192.Net_172_16_0_0",
            "start_time": 1640866550800,
            "status": "OPEN",
            "username_count": 2
          }
        ]
      }
      
      INFO[0016] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug run < tests/start_ariel_search.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: start_ariel_search
      AQL provided: Select * from events last 10 MINUTES
      HTTP/1.1" 201 None
      {
        "data": {
          "completed": false,
          "compressed_data_file_count": 0,
          "compressed_data_total_size": 0,
          "cursor_id": "fa69ede5-7e6b-4ed9-b8bb-368f80ee3b8d",
          "data_file_count": 0,
          "data_total_size": 0,
          "desired_retention_time_msec": 86400000,
          "index_file_count": 0,
          "index_total_size": 0,
          "processed_record_count": 0,
          "progress": 0,
          "progress_details": [],
          "query_execution_time": 0,
          "query_string": "Select * from events last 10 MINUTES",
          "record_count": 0,
          "save_results": false,
          "search_id": "fa69ede5-7e6b-4ed9-b8bb-368f80ee3b8d",
          "size_on_disk": 0,
          "status": "WAIT",
          "subsearch_ids": []
        }
      }
      
      INFO[0018] Running command:  docker run --rm -i rapid7-sacumen/ibm_qradar:1.0.0 --debug run < tests/update_offense.json 
      rapid7-sacumen/IBM QRadar:1.0.0. Step name: update_offense
      Offense ID provided: 33
      assigned_to provided: 
      closing_reason_id provided: 
      follow_up provided: False
      protected provided: False
      status provided: OPEN
      {
        "data": {
          "categories": [
            "Command Execution",
            "Input Validation Exploit",
            "Misc Exploit"
          ],
          "category_count": 3,
          "credibility": 3,
          "description": "Multiple Exploit/Malware Types Targeting a Single Destination\n containing Command Injection\n",
          "destination_networks": [
            "other"
          ],
          "device_count": 2,
          "domain_id": 0,
          "event_count": 4,
          "first_persisted_time": 1640155213000,
          "flow_count": 0,
          "follow_up": false,
          "id": 33,
          "inactive": true,
          "last_persisted_time": 1641908218000,
          "last_updated_time": 1640155212603,
          "local_destination_address_ids": [],
          "local_destination_count": 0,
          "log_sources": [
            {
              "id": 23312,
              "name": "Virsec Security Platform",
              "type_id": 4013,
              "type_name": "VirsecSecurityPlatformCustom"
            },
            {
              "id": 63,
              "name": "Custom Rule Engine-8 :: ip-172-31-34-93",
              "type_id": 18,
              "type_name": "EventCRE"
            }
          ],
          "magnitude": 3,
          "offense_source": "49.206.13.49",
          "offense_type": 1,
          "policy_category_count": 0,
          "protected": false,
          "relevance": 0,
          "remote_destination_count": 1,
          "rules": [
            {
              "id": 100113,
              "type": "CRE_RULE"
            }
          ],
          "security_category_count": 3,
          "severity": 9,
          "source_address_ids": [
            18
          ],
          "source_count": 1,
          "source_network": "other",
          "start_time": 1640155212500,
          "status": "OPEN",
          "username_count": 0
        }
      }

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes

``` 
[[33m*(B[m] Use ``make menu`` for available targets
[[33m*(B[m] Including available Makefiles: ../../tools/Makefiles/Colors.mk ../../tools/Makefiles/Helpers.mk
--
[[33m*(B[m] Running validators
[[33m*[0m] Validating plugin at .

[[33m*[0m] [1mRunning Integration Validators...[0m
[[33m*[0m] Executing validator HelpValidator
[[33m*[0m] Executing validator ChangelogValidator
[[33m*[0m] Executing validator CloudReadyConnectionCredentialTokenValidator
[[33m*[0m] Executing validator RequiredKeysValidator
[[33m*[0m] Executing validator UseCaseValidator
[[33m*[0m] Executing validator SpecPropertiesValidator
[[33m*[0m] Executing validator SpecVersionValidator
[[33m*[0m] Executing validator FilesValidator
[[33m*[0m] Executing validator TagValidator
[[33m*[0m] Executing validator DescriptionValidator
[[33m*[0m] Executing validator TitleValidator
[[33m*[0m] Executing validator VendorValidator
[[33m*[0m] Executing validator DefaultValueValidator
[[33m*[0m] Executing validator IconValidator
[[33m*[0m] Executing validator RequiredValidator
[[33m*[0m] Executing validator VersionValidator
[[33m*[0m] Executing validator DockerfileParentValidator
[[33m*[0m] Executing validator ProfanityValidator
[[33m*[0m] Executing validator AcronymValidator
[[33m*[0m] Executing validator JSONValidator
[[33m*[0m] Executing validator OutputValidator
[[33m*[0m] Executing validator RegenerationValidator
[[33m*[0m] Executing validator HelpInputOutputValidator
[[33m*[0m] Executing validator SupportValidator
[[33m*[0m] Executing validator RuntimeValidator
[[33m*[0m] Executing validator VersionPinValidator
[[33m*[0m] Executing validator EncodingValidator
[[33m*[0m] Executing validator ExampleInputValidator
[[33m*[0m] Executing validator CloudReadyValidator
[[33m*[0m] Executing validator SupportedVersionValidator
[[33m*[0m] Executing validator UnapprovedKeywordsValidator
[33mWARNING: Unsupported keywords found: offense, ariel. The following keywords will not be searchable by the Extension Library. Please remove or update the invalid keywords from the keywords array in the plugin.spec.yaml file.
[[33m*[0m] Executing validator HelpExampleValidator
[[33m*[0m] Executing validator Version Increment Validator
[[33m*[0m] [1mPlugin successfully validated![0m

----
[[33m*[0m][1m Total time elapsed: 2918.183ms[0m

```
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

![image](https://user-images.githubusercontent.com/88305288/152302220-4131efc5-feb3-4204-a860-c5640a52396e.png)

![image](https://user-images.githubusercontent.com/88305288/152303812-a6186e73-6af1-4c65-a92a-b12a617f3c7f.png)

![image](https://user-images.githubusercontent.com/88305288/152303857-d9cf6fce-764f-4274-af2b-0d0934c64ed3.png)

### CODE REVIEW

File: plugins/ibm_qradar/help.md [hunk 1]
```diff
@@ -65,8 +65,9 @@ Example input:
 ```
 {
```
Comment (blaxminarayan-r7) [inline plugins/ibm_qradar/help.md:66]: Can you please add an example with subfield. That will be very useful.

**Review by blaxminarayan-r7 — CHANGES_REQUESTED** [review]

File: plugins/ibm_qradar/help.md (hunk 1 above)
Comment (hiren-sacumen) [reply plugins/ibm_qradar/help.md:66]: Added for possible actions. This action does not have sub fields

File: plugins/ibm_qradar/icon_ibm_qradar/util/utils.py [hunk 2]
```diff
@@ -68,12 +67,12 @@ def validate_query_range(query_range: str) -> (bool, str):
...
         start = int(start.strip())
         end = int(end.strip())
-        if start < 0:
-            return False, "Range should start from 0 or greater than 0 "
-        if end < 0:
-            return False, "Range should start end 0 or greater than 0"
+        if start < 1:
+            return False, "Range should start from 1 or greater than 1 "
+        if end < 1:
+            return False, "Range should start end 1 or greater than 1"
```
Comment (mberezin-r7) [inline plugins/ibm_qradar/icon_ibm_qradar/util/utils.py:73]:
```suggestion
        if start < 1:
            return False, "Start of range should be >= 1"
        if end < 1:
            return False, "End of range should be >= 1"
```

File: plugins/ibm_qradar/plugin.spec.yaml [hunk 3]
```diff
@@ -797,20 +799,21 @@ actions:
...
-        description: Restrict the number of returned elements to a range, eg. 0-10, with 0 being the first index
+        description: Paging requests are specified with the Range parameter. E.g Example Default Range returns the first 50 records Custom Range like 1-5 will Returns the first 5 records and Range like 6-10 Return the 6th to 10th records
+        default: 1-50
         example: 1-2
       filter:
         title: Filter
         type: string
         required: false
-        description: Restrict the elements in a list based on the contents of various fields
+        description: Restrict the elements in a list based on the contents of various fields. E.g  id=55 and is_deleted = false
```
Comment (mberezin-r7) [inline plugins/ibm_qradar/plugin.spec.yaml:809]:
```suggestion
        description: Restrict the elements in a list based on the contents of various fields. E.g. id=55 and is_deleted = false
```

File: plugins/ibm_qradar/plugin.spec.yaml [hunk 4]
```diff
@@ -900,20 +903,21 @@ actions:
         title: Range
         type: string
         required: false
-        description: Restrict the number of returned elements to a range, eg. 0-10, with 0 being the first index
+        description: Paging requests are specified with the Range parameter. E.g Example Default Range returns the first 50 records Custom Range like 1-5 will Returns the first 5 records and Range like 6-10 Return the 6th to 10th records
```
Comment (mberezin-r7) [inline plugins/ibm_qradar/plugin.spec.yaml:906]:
```suggestion
        description: Paging requests are specified with the Range parameter. E.g. the example default range returns the first 50 records, a custom range of 6-10 returns the 6th to 10th records
```

File: plugins/ibm_qradar/plugin.spec.yaml [hunk 5]
```diff
@@ -900,20 +903,21 @@ actions:
...
-        description: Restrict the elements in a list based on the contents of various fields
+        description: Restrict the elements in a list based on the contents of various fields. E.g  id=55 and username = admin
         example: id=100
       fields:
         title: Fields
         type: string
         required: false
-        description: Specify which fields you would like to get back in the response. Fields that are not named are excluded. Specify subfields in brackets and multiple fields in the same object are separated by commas
-        example: id
+        description: Specify the list of fields to be returned in the response. Specify the subfields in brackets (). Multiple fields in the same object must be comma separated. Sample fields to filter are  id, create_time, username, note_text. More information about the fields can be found in plugin documentation
```
Comment (mberezin-r7) [inline plugins/ibm_qradar/plugin.spec.yaml:919]:
```suggestion
        description: Specify the list of fields to be returned in the response. Specify the subfields in parentheses. Multiple fields in the same object must be comma separated. Sample fields to filter are id, create_time, username, note_text. More information about the fields can be found in plugin documentation
```

File: plugins/ibm_qradar/plugin.spec.yaml [hunk 6]
```diff
@@ -940,8 +944,8 @@ actions:
...
```
Comment (mberezin-r7) [inline plugins/ibm_qradar/plugin.spec.yaml:947]:
```suggestion
        description: Specify the list of fields to be returned in the response. Specify the subfields in parentheses. Multiple fields in the same object must be comma separated. Sample fields to filter are id, create_time, username, note_text. More information about the fields can be found in plugin documentation
```

File: plugins/ibm_qradar/plugin.spec.yaml [hunk 7]
```diff
@@ -968,8 +972,8 @@ actions:
...
```
Comment (mberezin-r7) [inline plugins/ibm_qradar/plugin.spec.yaml:975]:
```suggestion
        description: Specify the list of fields to be returned in the response. Specify the subfields in parentheses. Multiple fields in the same object must be comma separated. Sample fields to filter are id, create_time, username, note_text. More information about the fields can be found in plugin documentation
```

File: plugins/ibm_qradar/plugin.spec.yaml [hunk 8]
```diff
@@ -985,25 +989,26 @@ triggers:
         title: Range
         type: string
         required: false
-        description: Restrict the number of returned elements to a range, eg. 0-10, with 0 being the first index
+        description: Paging requests are specified with the Range parameter. E.g Example Default Range returns the first 50 records Custom Range like 1-5 will Returns the first 5 records and Range like 6-10 Return the 6th to 10th records
```
Comment (mberezin-r7) [inline plugins/ibm_qradar/plugin.spec.yaml:992]:
```suggestion
        description: Paging requests are specified with the Range parameter. E.g. the example default range returns the first 50 records, a custom range of 6-10 returns the 6th to 10th records
```

File: plugins/ibm_qradar/plugin.spec.yaml [hunk 9]
```diff
@@ -985,25 +989,26 @@ triggers:
...
         example: 1-2
+        default: 1-50
       filter:
         title: Filter
         type: string
         required: false
-        description: Restrict the elements in a list based on the contents of various fields
+        description: Restrict the elements in a list based on the contents of various fields. E.g  id=55 and follow_up = false
```
Comment (mberezin-r7) [inline plugins/ibm_qradar/plugin.spec.yaml:999]:
```suggestion
        description: Restrict the elements in a list based on the contents of various fields. E.g. id=55 and follow_up = false
```

File: plugins/ibm_qradar/plugin.spec.yaml [hunk 10]
```diff
@@ -985,25 +989,26 @@ triggers:
...
         example: id=100
       fields:
         title: Fields
         type: string
         required: false
-        description: Specify which fields you would like to get back in the response. Fields that are not named are excluded. Specify subfields in brackets and multiple fields in the same object are separated by commas
-        example: id
+        description: Specify the list of fields to be returned in the response. Specify the subfields in brackets (). Multiple fields in the same object must be comma separated. Sample fields to filter are  id, assigned_to, inactive, offense_source,  offense_type, rules(id), protected, follow_up, status, domain_id, rules, log_sources etc. More information about the fields can be found in plugin documentation
```
Comment (mberezin-r7) [inline plugins/ibm_qradar/plugin.spec.yaml:1005]:
```suggestion
        description: Specify the list of fields to be returned in the response. Specify the subfields in parentheses. Multiple fields in the same object must be comma separated. Sample fields to filter are id, assigned_to, inactive, offense_source, offense_type, rules(id), protected, follow_up, status, domain_id, rules, log_sources. More information about the fields can be found in plugin documentation
```

File: plugins/ibm_qradar/plugin.spec.yaml [hunk 11]
```diff
@@ -985,25 +989,26 @@ triggers:
...
+        example: id, rules(id)
       sort:
         title: Sort
         type: string
         required: false
-        description: Sorting strategy for list elements
+        description: Apply sorting order to result sets i.e. +id which indicates sort the result in ascending order of id value
```
Comment (mberezin-r7) [inline plugins/ibm_qradar/plugin.spec.yaml:1011]:
```suggestion
        description: Apply sorting order to result sets, e.g. +id to sort the result in ascending order of id value
```

**Review by mberezin-r7 — COMMENTED** [review]
Minor grammatical changes, please apply comments and then regenerate the plugin

**Review by mberezin-r7 — APPROVED** [review]

**Review by blaxminarayan-r7 — APPROVED** [review]

**Pull request merged**